repository-data/application/src/main/resources/hcm-config/namespaces/hippostd/html.yaml

```diff
@@ -0,0 +1,5 @@
+---
+definitions:
+  config:
+    /hippo:namespaces/hippostd/html/editor:templates/_default_:
+      ckeditor.config.overlayed.json: '{removePlugins: ''stylescombo,sourcearea,codemirror,youtube,table,tableresize,tableselection,tabletools,specialchar,image,image2,textselection,toolbar,link,indent,indentblock,indentlist,justify,hippopicker,elementspath''}'
```

Comment:
@giacomolm - I think this will remove it from all HTML editors, right?

If true, can you let @ngta1 know how to disable it only for some content types?

Reply:
It can be global and field-specific.  Its currently global so will affect all RTE controls.  I can move that line to the individual field if required?  We use the RTE for other doc types like the About pages.

Reply:
this change will affect "digital website" and "NIL" streams as well.

Might be worth checking with @giacomolm if you can define global "profiles" so you don't have to copy whole config to each content type.

Reply:
I would suggest locally to the specific document type, unless it's a global requirement. We did something similar [here|https://github.com/NHS-digital-website/hippo/blob/master/repository-data/application/src/main/resources/hcm-config/namespaces/website/service.yaml]

Reply:
ace!